web/elm/src/Build/StepTree.elm

```diff
@@ -747,7 +747,7 @@ viewTimestamp hl id ( line, date ) =
         ]
         [ case date of
             Just date ->
-                Html.td [ class "timestamp", attribute "data-timestamp" (Date.Format.format "%H:%M:%S" date) ] []
```

Comment:
Woah, I don't think this is what we want. This will put a full timestamp including the year, month, and day on every single line of build output. Doesn't that seem like a bit much, especially on narrower screens?

These timestamps are designed for seeing where time is spent in between individual logs, and also for deep-linking into output. I really don't think you want them to have this much information. I thought this was going to be about the start/finish timestamps in the header:

![screenshot from 2019-01-29 10-48-08](https://user-images.githubusercontent.com/1880/51920442-65781f00-23b3-11e9-8b21-ae098ae7c5e6.png)

Reply:
Let me put a screen shot here:

![concourseui-show-date](https://user-images.githubusercontent.com/18585861/51946514-b7c14a80-245c-11e9-9d61-da6bb090bdcd.png)

@vito I think YMD formatted date is not a problem to nowadays wide screen, and it more fits user's reading habit. Actually this issue was raised by our user (VMC team, the biggest and most important internal customer).

I agree the banner `started` and `finished` could be changed as well. We may help make a PR as well.